automation/test.sh

```diff
@@ -481,6 +481,19 @@ spec:
 EOF
 fi
 
+ksm_available_nodes_count=0
```

Comment:
By misconfiguration I mean a case where I forgot to enable KSM in the `project-infra`. With this, you would skip the tests but that would be wrong because I would rather get feedback, "You forgot about KSM!". Therefore all I am asking is rather not have this and move the responsibility of skip to elsewhere.

Reply:
Well, this is not related to enable ksm, but if it is available.
Are you suggesting to switch bask to `Skip()` inside the tests?

Reply:
I think KSM is always compiled in the kernel for CentOS, so not having it would be pretty hard, that said here probably anything works, even just always adding the KSM ginkgo label without detecting anything.

Reply:
>Are you suggesting to switch bask to Skip() inside the tests?

Not really. I am suggesting to not do any auto-detections. We should stick to what we do now, for example, add skip for lanes.
It is the same as you would be detecting if there is KVM or GPU... You never know if you did provision the right hardware or if the components that expose the resources just did not die. It is better to fail than a false skip.

Reply:
Got it, thanks!